### DISCUSSION TITLE
从 valine 迁移到自建数据库的函数

### DISCUSSION THREAD

**Discussion by XiongLiding** [walinejs/waline#64] [Q&A] [answered]
看到文档里有个从 valine 迁移到 cloudbase 的转换脚本：
https://waline.js.org/migration.html#%E8%BF%81%E7%A7%BB%E8%87%B3-cloudbase

我自己写了个转换到 CSV 格式的函数，可以方便导入到自建数据库：
https://gist.github.com/XiongLiding/fc70b644c54d0c2bc1a6ed788469a93a

**Comment by lizheming** [comment]
赞一个，SQL 类的应该都可以用这个，我之后加到文档中，非常感谢！

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/migration/tool.html 已将该逻辑合并到官方文档中来了，非常感谢~

**Comment by lizheming** [comment]
https://gist.github.com/XiongLiding/fc70b644c54d0c2bc1a6ed788469a93a#file-valinecommenttocsv-js-L22
@XiongLiding 这块你为什么要扔掉第一条数据？

**Reply by XiongLiding** [reply]
应该是我当时逻辑混乱了，脑子里想着 CSV 就顺手把第一行给丢了 😓
